src/pypromice/qc/persistence.py

```diff
@@ -14,13 +14,14 @@
 
 logger = logging.getLogger(__name__)
 
+# period is given in hours, 2 persistent 10 min values will be flagged if period < 0.333
 DEFAULT_VARIABLE_THRESHOLDS = {
-    "t": {"max_diff": 0.0001, "period": 2},
-    "p": {"max_diff": 0.0001, "period": 2},
-    'gps_lat_lon':{"max_diff": 0.000001, "period": 12}, # gets special handling to remove simultaneously constant gps_lat and gps_lon
-    'gps_alt':{"max_diff": 0.0001, "period": 12},
-    't_rad':{"max_diff": 0.0001, "period": 2},
-    "rh": {"max_diff": 0.0001, "period": 2}, # gets special handling to allow constant 100%
+    "t": {"max_diff": 0.0001, "period": 0.3},
```

Comment:
I think the period length fro t and p was thought to be 2 hours and not 20 minutes.

Reply:
Ah, yes, because it was meant to handle transmissions.
But now that we are talking about it, I suggest that we keep it at period = 0.3 hours because we know that the loggers were, during a time, wrongly programed to report the last valid number instead of NaN. So we cannot differentiate a good measurement with the same value as the previous one (rather unlikely) from a missing observation filled with the last valid value.

Reply:
Maybe it is necessary to define configurations specifically for tx, raw, hourly and 10 minutes data.

p_u and t_u only have a single decimal precision in the tx data files hence I think there is a significant risk of filtering out valid data.

Reply:
> p_u and t_u only have a single decimal precision in the tx data files hence I think there is a significant risk of filtering out valid data.

I didn't know that, good point! I switch back to period = 2 hours until we have a specific handling for raw data.